gradoop-common/src/test/java/org/gradoop/common/model/impl/pojo/GraphElementTest.java

```diff
@@ -0,0 +1,39 @@
+package org.gradoop.common.model.impl.pojo;
+
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.id.GradoopIdSet;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.junit.Test;
+import org.mockito.Mockito;
```

Comment:
You don't need this import since you are using static imports below.

Reply:
done